src/containers/login/Login.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import { LoginForm } from '../../components/loginForm/LoginForm';
-import { NewUserForm } from '../../components/newUserForm/NewUserForm';
+import { checkCredentials, getUser } from '../../data';
+
+export const Login = ({
+  setLoggedIn,
+  setUser,
+  newPatient
+}) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const submitCredentials = () => {
+    if(checkCredentials(username, password)){
```

Comment:
We should stress in the challenge's description that this is not how you would handle passwords and authentications in a real project.